lib/theme_check/remote_asset_file.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require "net/http"
+require "pathname"
+require "zlib"
+
+module ThemeCheck
+  class RemoteAssetFile
+    class << self
+      def cache
+        @cache ||= {}
+      end
+
+      def from_src(src)
+        key = uri(src).to_s
+        cache[key] = RemoteAssetFile.new(src) unless cache.key?(key)
+        cache[key]
+      end
+
+      def uri(src)
+        URI.parse(src.sub(%r{^//}, "https://"))
+      end
+    end
+
+    def initialize(src)
+      @uri = RemoteAssetFile.uri(src)
+    end
+
+    def content
+      @content ||= Net::HTTP.get(@uri)
+    end
+
+    def gzipped_size
+      @gzipped_size ||= Zlib.gzip(content).size
```

Comment:
I don't think we do. I used it this way for consistency with AssetFile (also, what if the server doesn't gzip?)

Reply:
If we set the same header as Chrome (`Accept-Encoding: gzip, deflate, br`), the server will just pick the best one. It will be the same bytes as the browser sees it, which it what you want to measure.